src/main/java/com/example/pizzapp/repositories/ProductRepository.java

```diff
@@ -0,0 +1,20 @@
+package com.example.pizzapp.repositories;
+
+
+import com.example.pizzapp.models.Product;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+import java.util.Optional;
+
+@Repository
+public interface ProductRepository extends JpaRepository<Product, Long> {
+
+ Optional<Product> findById(Long id);
+
+ Optional<Product> findByName(String name);
+
+ List<Product> findByCategoryId(Long id);
```

Comment:
```suggestion
 List<Product> findAllByCategoryId(Long categoryId);
```
И еще в этом файле форматирование сбилось